index.html

```diff
@@ -2502,23 +2506,20 @@ <h2>The <dfn>NFCAccept</dfn> enum</h2>
 
       <section><h3>Creating a Web NFC Id</h3>
       <p>
-        To <dfn>create a <a>Web NFC Id</a></dfn> given <var>url</var>,
+        To <dfn>create a <a>Web NFC Id</a></dfn> given <a>URL path</a> <var>urlPath</var>,
         run these steps:
         <ol>
           <li>
-            Let <var>id</var> be the <a>serialized origin</a> of the
-            <a>browsing context</a>, appended with <var>url</var>.
+            Let <var>id</var> be the <a>serialized origin</a> of the current
+            <a>browsing context</a>, append <var>urlPath</var> to <var>id</var>.
```

Comment:
These appear to be two separate operations, which should be separate steps, not a comma-joined single sentence.

Reply:
OK, I will separate them.